src/java/org/apache/cassandra/locator/DynamicEndpointSnitch.java

```diff
@@ -224,6 +226,11 @@ private <C extends ReplicaCollection<? extends C>> C sortedByProximityWithBadnes
         return replicas;
     }
 
+    private static double defaultStore(InetAddressAndPort target)
+    {
+        return USE_SEVERITY ? getSeverity(target) : 0.0;
```

Comment:
not sure if this change should also be behind a config flag... this is a change in behavior if you set severity... this is done to make this patch work in a stable manner, but outside of decom severity is used to limit traffic, yet we had different code paths to ignore it (such as reset every 10m).  By defaulting to severity we always respect the operators intent